src/app/shared/modules/workflow.module.ts

```diff
@@ -105,6 +106,7 @@ import { getTooltipConfig } from './../tooltip';
   providers: [
     { provide: TooltipConfig, useFactory: getTooltipConfig },
     DateService,
+    BioschemaService,
```

Comment:
Throw this into the entryModule instead of both workflow and container.

Reply:
moved to entry.module.ts